content/3_Silver/Flood_Fill.mdx

```diff
@@ -216,17 +216,23 @@ And we’ll want to input the grid as follows:
 <CPPSection>
 
 ```cpp
-int grid[MAXN][MAXM];
-int n, m;
-
-int main(){
-	cin >> n >> m;
-	for(int i = 0; i < n; i++){
-		for(int j = 0; j < m; j++){
-			cin >> grid[i][j];
+#include <iostream>
+
+using namespace std;
+
+constexpr int MAX_N = 1000;
```

Comment:
why constexpr over const?

Reply:
...Why not?

Reply:
theres no point

Reply:
The cost imposed by using `constexpr` (readers possibly having to search up what that means) outweighs the benefits which are practically none in this program.

Reply:
`constexpr` being a straight extension of `const` makes it pretty straightforward for readers to understand.

Reply:
but again there's no point so stick with what's everyone is used to